src/components/styles/ProjectPreview.styled.ts

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components'
+
+export const PreviewText = styled.div<{ hasAnimation: boolean }>`
+  font-size: 0.9rem;
+  color: ${(props) => props.theme.palette.common.white};
+  @keyframes fadeIn {
+    0% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+  ${(props) => {
+    if (props.hasAnimation) {
+      return `
+      animation-name: fadeIn;
+      animation-delay: 1s;
+      animation-duration: 1.5s;
+      animation-fill-mode: both; 
+      `
+    }
+    return `
+      animation-name: none;
+    `
+  }}
+`
+export const PreviewTitle = styled.div<{ fontSize?: string }>`
+  width: fit-content;
+  justify-content: center;
+  align-items: center;
+  color: ${(props) => props.theme.palette.common.white};
+  font-size: ${(props) => props.fontSize};
```

Comment:
since fontSize is optional, should there be a default size?

Reply:
4df3501bc050c8f6d3a80c59308608abbc6a60bd